pkg/querier/error_translate_queryable.go

```diff
@@ -62,9 +62,8 @@ func TranslateToPromqlAPIError(err error) error {
 			if code >= 400 && code < 500 {
 				// Return directly, will be mapped to 422
 				return err
-			} else if code >= 500 && code < 599 {
```

Comment:
Hmm, so this condition was entirely redundant before?

Reply:
We map everything that's not handled into `promql.ErrStorage{Err: err}`.

Reply:
@56quarters yes, it was redundant, as it gets handled after the `if ok` block.